src/zoo.cpp

```diff
@@ -0,0 +1,184 @@
+#include "..\include\zoo.h"
+#include <string>
+#define ROWS 20
+#define COLS 40
+        Zoo::Zoo()
+        {
+            createAnimal('L', "Simba");
+            createAnimal('L', "Mufasa");
+            createAnimal('M', "Monkey");
+            createAnimal('G', "Akka");
+            createAnimal('G', "Morten");
+        }
+        void Zoo::run()
+        {
+            {
+                printAllAnimals();
+                int index;
+                cout << "Type 'help' to see the list of commands.\n";
+                string command;
+                while (true) {
+                    cout << "> ";
+                    getline(cin, command);
+                    if (command == "stop") {
```

Comment:
it would be better if you a function to handle the commands, function run is too long